client/pages/debug/components.vue

```diff
@@ -0,0 +1,62 @@
+<script lang="ts" setup>
+</script>
+
+<template>
+  <div class="grid grid-cols-1 gap-3">
+    <div class="p-card p-5 flex flex-col gap-3">
+      <h2 class="text-2xl">
+        Buttons
+      </h2>
+      <div class="flex gap-2">
+        <PButton size="small" label="Primary" />
+        <PButton size="small" label="Secondary" severity="secondary" />
```

Comment:
Looks weird 

Secondary dark == surface, it's kinda weird no ?

![image](https://github.com/Black-Kamelia/Hedera/assets/46132319/dde98534-f845-48e1-8685-30085904463b)